### DISCUSSION TITLE
Please support vcpkg

### DISCUSSION THREAD

**Discussion by LiuPeiqiCN** [libvips/libvips#2743] [Ideas]
vcpkg is C++ Library Manager for Windows, Linux, and MacOS and supports 1916 libraries now, vcpkg also support meson build system.

**Comment by jcupitt** [comment]
Hi @liupeiqiHN,

I don't use it myself, so I won't do the work. But of course I'd be happy to help anyone who wants to volunteer to make and maintain a libvips package.
